crates/ruff_python_formatter/src/statement/suite.rs

```diff
@@ -71,7 +71,8 @@ impl FormatRule<Suite, PyFormatContext<'_>> for FormatSuite {
         let source = f.context().source();
         let source_type = f.options().source_type();
 
-        let f = &mut WithNodeLevel::new(node_level, f);
+        let f = WithNodeLevel::new(node_level, f);
+        let f = &mut WithIndentLevel::new(f.context().indent_level().next(), f);
```

Comment:
That looks too easy.

Reply:
That's what I thought. Hah.

And I had to do a little trickery to make this work. In particular, asking for a `DerefMut<Target=impl Buffer>` instead of an `impl Buffer` in order to make it composable with `WithNodeLevel`. I pondered for a bit on a better design, but there were multiple choices and it wasn't obvious to me which path to go.